### DISCUSSION TITLE
Move "Adjust pitch" to Nightcore/Daycore

### DISCUSSION THREAD

**Discussion by agatemosu** [ppy/osu#25249] [Ideas] [answered]
I don't see the point in having that option in Double Time and Half Time, and I think it would make sense for them to be in Nightcore and Daycore

edit: ok maybe 1,01 with pitch adjust in nightcore is not good

**Comment by bdach** [comment, marked as answer]
i swear there is no winning with you people. the adjust pitch toggle was requested by several people over the years and now this?

do you understand this would lead to the possibility of having plays set to nightcore have background music that _does not sound like nightcore_? might as well delete the mod at this point.

**Reply by rszyma** [reply]
I've already seen people question the point of having NC mod when we can now have nightcore with pitch shift enabled on DT. The only difference now between DT 1.5x + pitch shift and NC 1.5x is the additinal drums fx, and that alone is not defining for nightcore genre; the pitch shift is. 

> might as well delete the mod at this point

Actually, arguably this is a good idea. NC could be deleted as a standalone mod and another toggable option could be added to DT to enable the drums fx that NC has. Also, maybe the DT would become labeled as NC on leaderboards once user ticks "enable NC drums fx" option (and/or "pitch shift" option). This would also be compatible with existing scores with NC.

**Reply by bdach** [reply]
> the only difference now between DT 1.5x + pitch shift and NC 1.5x is the additinal drums fx

no, it is not. nightcore always forces 1.5x frequency adjustment, and then applies a tempo adjustment on top of it to follow the user's selected bpm. the frequency / pitch with nightcore mod is _constant_. it is not constant with double time with the toggle.
